web/db/migrations/20260818090000_coderouter_session_accounts/migration.sql

```
@@ -0,0 +1,19 @@
+  "account_id" uuid NOT NULL
+    REFERENCES "coderouter_accounts" ("id") ON DELETE CASCADE,
+  "created_at" timestamp with time zone DEFAULT now() NOT NULL,
+  "last_seen_at" timestamp with time zone DEFAULT now() NOT NULL,
+  CONSTRAINT "coderouter_session_accounts_pkey"
+    PRIMARY KEY ("team_id", "provider", "session_key"),
+  CONSTRAINT "coderouter_session_accounts_provider_check"
+    CHECK ("provider" IN ('codex', 'opencode-go'))
+);
+
+CREATE INDEX "coderouter_session_accounts_account_idx"
+  ON "coderouter_session_accounts" ("account_id");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)